src/components/support/Product.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <div class="product">
```

Comment:
If you add the class `"box"`, this will give it a white background with rounded border, and will also style the title (if `h4`) correctly